views/random-beer-page.hbs

```diff
@@ -0,0 +1,23 @@
+<div>
+    {{#each beersarray}}
+
+
+<div class="card mb-3" style="max-width: 540px;">
+    <div class="row g-0">
+        <div class="col-md-4">
+            <img src="{{image_url}}" class="img-fluid rounded-start" alt="{{name}}">
+        </div>
+        <div class="col-md-8">
+            <div class="card-body">
+                <h5 class="card-title">{{name}}</h5>
+                <p class="card-text">{{description}}</p>
+                <p class="card-text"><small class="text-muted">{{tagline}}</small></p>
+                <class="card-text">{{food_pairing}}</p>
```

Comment:
aunque te lo pinte bien, hubiese estado chulo meter otro #each ya que esto es otro array y haberlo metido en una etiqueta <li>
<ul>
                        {{#each food_pairing}}
                        <li>{{this}}</li>
                        {{/each}}
                    </ul>